tests/testing.h

```diff
@@ -36,7 +36,7 @@
 #include <sstream>
 #include <functional>
 
-#ifdef _WIN32
+#if (defined _WIN32) || (defined SDL2PP_XCODE_GENERATED)
```

Comment:
I'd rather fix it by switching to catch then. You could still submit .gitignore changes in separare PR.